javascript-practice/src/styles/layouts/index.css

```diff
@@ -1,2 +1,4 @@
 @import "./container.css";
-@import "./header.css";
\ No newline at end of file
+@import "./header.css";
+@import "./toast.css";
+@import "./error.css";
```

Comment:
Pls add empty at the end of file. Btw, I comment for issue on this. But pls check all your code for similar issue